spk/homeassistant/Makefile

```diff
@@ -11,16 +11,16 @@ BUILD_DEPENDS += cross/pyephem cross/netifaces cross/psutil
 
 SPK_DEPENDS = "python3>=3.4.1-4:git"
 
-DEPENDS = cross/busybox cross/nmap #for python-libnmap
+DEPENDS = cross/$(SPK_NAME) cross/busybox cross/nmap #for python-libnmap
 
 MAINTAINER = SynoCommunity
-DESCRIPTION = Home Assistant is an open-source home automation platform running on Python 3. Track and control all devices at home and automate control.
+DESCRIPTION = "Home Assistant is an open-source home automation platform running on Python 3. Track and control all devices at home and automate control. The default password is empty, just click \"Login\".  Edit your settings via ssh in the configuration.yaml file. This is a beta package. Hue and Chrome are tested. Other modules might work, please test them and try to contribute if something does not work."
```

Comment:
Please keep the description limited to the package, so something like this: `DESCRIPTION = "Home Assistant is an open-source home automation platform running on Python 3. Track and control all devices at home and automate control. The default password is empty, just click \"Login\".`

The rest of the text does not belong here. If official documentation is available explaining what you need, use that (SUPPORTURL). Alternatively, use a help page (HELPURL) to point to a help page or wiki page explaining these details. Realistically, the only necessary sentence is this one: `Edit your settings via ssh in the configuration.yaml file`, the rest is implied as it's a beta package.